nb-adapters/adapter-milvus/pom.xml

```diff
@@ -52,26 +52,21 @@
             <artifactId>protobuf-java-util</artifactId>
             <version>3.25.3</version>
         </dependency>
-
-<!--        <dependency>-->
-<!--            <groupId>org.apache.logging.log4j</groupId>-->
-<!--            <artifactId>log4j-core</artifactId>-->
-<!--            <version>2.23.1</version>-->
-<!--        </dependency>-->
-
         <dependency>
             <groupId>io.milvus</groupId>
             <artifactId>milvus-sdk-java</artifactId>
             <version>2.4.1</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/ch.qos.reload4j/reload4j replaces log4j 1.X-->
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <groupId>ch.qos.reload4j</groupId>
```

Comment:
We don't want to add another logging dependency here if we can help it.
The interaction between all the modules regarding logging is complex. The strategy for managing it thus far is to 1) Standardize as much as possible on log4j2.
2) Add whatever slf4j stubs or bridging modules needed to handle per-module dependencies.

I'll look closer at this module's dependencies and understand it before making further recommendations.

Reply:
Milvus is using the ancient unsupported Log4j 1.2.X and ch.qos.reload4j is a maintained fork of Log4J 1.2.X maintained by the original log4j author. Talk to Milvus about why they are doing this.

Reply:
I'll provide a `mvn dependency:tree` comparison

Reply:
It look like further study of https://www.slf4j.org/manual.html maybe required, but mostly because both cases have this: